chaoslib/service_chaos/svc_chaos.yml

```diff
@@ -0,0 +1,87 @@
+- block:
+    - name: Setup svc chaos infra
+      shell: >
+        kubectl create -f /chaoslib/service_chaos/svc_chaos_kube.yml 
+        -n {{ namespace }}
+      args:
+        executable: /bin/bash
+      register: result
+    
+    - name: Confirm that the svc chaos ds is running on all nodes
+      shell: >
+        kubectl get pod -l app=svc-chaos
+        --no-headers -o custom-columns=:status.phase
+        -n {{ namespace }} | sort | uniq
+      args: 
+        executable: /bin/bash
+      register: result
+      until: "result.stdout == 'Running'"
+      delay: 20
+      retries: 15
+
+    - name: Obtaining the application pod name using its label.
+      shell: >
+        kubectl get pods -n {{ app_ns }} -l {{ app_label }} | grep
+        -w Running | awk '{print $1}'
+      args:
+        executable: /bin/bash
+      register: app_pod
+
+    - name: Identify the application node
+      shell: >
+        kubectl get pod {{ app_pod.stdout }} -n {{ namespace }}
+        --no-headers -o custom-columns=:spec.nodeName
+      args: 
+        executable: /bin/bash
+      register: result 
+               
+    - name: Record the application node name 
+      set_fact:
+        app_node: "{{ result.stdout }}"
+ 
+    - name: Record the svc chaos pod on given app node 
+      shell: >
+        kubectl get pod -l app=svc-chaos -o wide 
+        -n {{ namespace }} | grep {{ app_node }} 
+        | awk '{print $1}'
+      args:
+        executable: /bin/bash
+      register: svc_chaos_pod
+
+    - name: do svc restart
+      shell: >
+        timeout 10 kubectl exec -it {{ svc_chaos_pod.stdout}} -n {{ namespace }} 
+        -- bash -c "systemctl stop kube-docker-monitor && systemctl stop kubelet-monitor && systemctl stop {{ svc_chaos }} && sleep {{ c_duration }} && systemctl start {{ svc_chaos }} && systemctl start kube-docker-monitor && systemctl start kubelet-monitor"
```

Comment:
`systemctl stop kube-docker-monitor && systemctl stop kubelet-monitor` are platform dependent. In AWS cluster(ubuntu image) these services are not available to perform `svc_chaos`. 
```
ubuntu@ip-10-0-1-79:~$  sudo systemctl stop kube-docker-monitor
Failed to stop kube-docker-monitor.service: Unit kube-docker-monitor.service not loaded.
ubuntu@ip-10-0-1-79:~$ sudo systemctl stop kubelet-monitor
Failed to stop kubelet-monitor.service: Unit kubelet-monitor.service not loaded.
```
Is it better to pass other services(kube-docker-monitor,kubelet-monitor) from utils like svc_chaos?

Reply:
good point @mittachaitu ! To find/arrive on the allied services to stop/start with docker - the test will need to be run on multiple platforms. We should be OK to take this for now IMO & conditionally pass the additional services to the svc_chaos utils when we have the details.